forte/data/extractors/subword_extractor.py

```diff
@@ -81,15 +86,21 @@ def extract(self, pack: DataPack, instance: Annotation) -> Feature:
             text = subword.text
             if not subword.is_first_segment:
                 text = '##' + text
-            if text in self.tokenizer.vocab:
-                data.append(self.tokenizer.vocab[text])
+            if self.vocab:
+                data.append(self.element2repr(text))
             else:
-                data.append(self.tokenizer.vocab['[UNK]'])
+                data.append(text)
```

Comment:
Are there cases we don't have a `vocab` at all and use the text?

Reply:
If not, we can simply throw exception if vocab is None, and use the repr all the time.

Reply:
OK